src/main/java/org/jenkinsci/test/acceptance/po/WorkflowJob.java

```diff
@@ -35,7 +40,60 @@ public WorkflowJob(Injector injector, URL url, String name) {
         super(injector, url, name);
     }
 
-    public final Control script = control("/definition/script");
+    public final Control script = new Control(this, "/definition/script") {
+        @Override
+        public void set(String text) {
+            try {
+                super.set(text);
```

Comment:
Not if you run the test against 1.580.x or 1.596, in which case 1.8 will not even be available.